### DISCUSSION TITLE
When 22.0.0 milestone is planned?

### DISCUSSION THREAD

**Discussion by admin-athena** [keycloak/keycloak#20627] [Q&A] [answered]
Can you please provide a rough ETA?

**Comment by benstpierre** [comment]
22 is going to be on Jakarta EE 10 right?

**Reply by oliveryasuna** [reply]
Further, is there a repo to try 22?

**Comment by vmuzikar** [comment, marked as answer]
Hello, we're releasing major/minor versions with approximately 3 months cadence. Last such release was 21.1 in April.

Keycloak 22 will be based on Jakarta. You can see WIP release notes [here](https://github.com/keycloak/keycloak/blob/main/docs/documentation/release_notes/topics/22_0_0.adoc).

You can try our [nightly releases](https://github.com/keycloak/keycloak/releases/tag/nightly) to get the idea of what's coming in 22.
